docs/app/pages/components/sections/input-controls/sliders/snippets/sample4.ts

```diff
@@ -51,7 +51,7 @@ export class SliderExampleComponent {
             }
         };
 
-        this.lowerValue.debounceTime(300)
+        this.lowerValue.debounceTime(500)
```

Comment:
How about updating on blur/enter? We can't really decide a limit on how slowly users will type, and this example will be impossible to use if one does not type that fast.